console/server/src/main/java/quarks/console/server/ServerUtil.java

```diff
@@ -45,31 +53,35 @@ private String getPath() {
      * @return a File object
      */
     private File getTopDirFilePath() {
+        String topDirProp = System.getProperty("top.dir.file.path");
```

Comment:
I think now is best.  It won't make converting any harder / will just get adjusted when we handle all other occurrences.